### PR TITLE
[FrameworkBundle] add support for prioritizing form type extension tags

### PR DESCRIPTION
**Pull request by dmaicher**
Doc PR for https://github.com/symfony/symfony/pull/19790

### CODE REVIEW

**Comment by HeahDude** [comment]
Looks good! Thanks :)

File: form/create_form_type_extension.rst [hunk 1]
```diff
@@ -128,6 +128,14 @@ The ``extended_type`` key of the tag is the type of field that this extension sh
 be applied to. In your case, as you want to extend the ``Symfony\Component\Form\Extension\Core\Type\FileType``
 field type, you will use that as the ``extended_type``.
 
+.. tip::
+
+    There is an optional tag attribute called ``priority``, which 
+    defaults to ``0`` and it controls the order in which the form  
+    type extensions are loaded (the higher the priority, the earlier 
+    an extension is loaded). This is useful when you need to guarantee 
+    that one extensions is loaded before or after another extension.
```
Comment (ogizanagi) [inline form/create_form_type_extension.rst]:
```diff
- that one extensions
+ that one extension
```

File: form/create_form_type_extension.rst [hunk 2]
```diff
@@ -128,6 +128,14 @@ The ``extended_type`` key of the tag is the type of field that this extension sh
 be applied to. In your case, as you want to extend the ``Symfony\Component\Form\Extension\Core\Type\FileType``
 field type, you will use that as the ``extended_type``.
 
+.. tip::
+
+    There is an optional tag attribute called ``priority``, which 
+    defaults to ``0`` and it controls the order in which the form
```
Comment (ogizanagi) [inline form/create_form_type_extension.rst]: `it` may be removed as it is redundant with `which` (IMHO)

File: form/create_form_type_extension.rst (hunk 1 above)
Comment (dmaicher) [inline form/create_form_type_extension.rst]: :+1:

File: form/create_form_type_extension.rst (hunk 2 above)
Comment (dmaicher) [inline form/create_form_type_extension.rst]: :+1:

**Comment by xabbuh** [comment]
We should add a `versionadded` directive for this.

Status: Reviewed

**Comment by javiereguiluz** [comment]
👍

**Comment by xabbuh** [comment]
Thank you @dmaicher.

**Pull request closed without merging**